src/lib/stream.rs

```diff
@@ -291,6 +352,7 @@ impl<'a> CodedInputStream<'a> {
         )
     }
 
+    #[cfg(target_endian = "big")]
```

Comment:
`big` and `little` should be on function, and at the end `.to_le()` function should be called to swap bytes on big-endian and do nothing on little-endian.